amgut/lib/util.py

```diff
@@ -169,8 +170,9 @@ def survey_vioscreen(survey_id):
     """Return a formatted text block and URL for the external survey"""
     tl = text_locale['human_survey_completed.html']
     embedded_text = tl['SURVEY_VIOSCREEN']
+    user_info = ag_data.get_person_info(survey_id)
     url = ("https://vioscreen.com/remotelogin.aspx?Key=%s&RegCode=KLUCB" %
-           url_escape(encrypt_key(survey_id)))
+           url_escape(encrypt_key(survey_id, user_info)))
```

Comment:
...should have simply scrolled a little further. Makes sense now, thanks. @squirrelo, it would still be useful if you could once over these particular changes though